operations/mimir-mixin/alerts/blocks.libsonnet

```diff
@@ -1,4 +1,32 @@
+local utils = import 'mixin-utils/utils.libsonnet';
+
 (import 'alerts-utils.libsonnet') {
+  local highVolumeLevel1BlocksQueried(histogram_type) = {
+    // Alert if there's sustained querying of level 1 blocks, which indicates the compactor
+    // is not keeping up and the store-gateway is serving blocks that aren't well compacted.
+    alert: $.alertName('HighVolumeLevel1BlocksQueried'),
+    local sum_by = [$._config.alert_aggregation_labels],
+    local range_interval = $.alertRangeInterval(10),
+    local job = $.jobMatcher($._config.job_names.store_gateway),
+    local numerator = utils.ncHistogramSumBy(utils.ncHistogramSumRate('cortex_bucket_store_series_blocks_queried', 'component="store-gateway",level="1",out_of_order="false",%(job)s' % job, rate_interval=range_interval, from_recording=false), sum_by),
```

Comment:
It seems like `cortex_bucket_store_series_blocks_queried` is a summary, so this will have no effect, can be reverted: https://github.com/grafana/mimir/blob/ec14e21eecc60c8fce9880ee966db8cadf8d41bd/pkg/storegateway/bucket_store_metrics.go#L106

Reply:
Good catch, thank you! I was just looking out for the `NativeHistogramBucketFactor` stuff and missed that it was actually a summary instead.